cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/VerifyBundle.kt

```diff
@@ -88,6 +88,7 @@ open class VerifyBundle @Inject constructor(objects: ObjectFactory) : DefaultTas
             verifier.setProperty(STRICT, strict.get().toString())
             verifier.verify()
             verifyImportPackage(verifier)
+            verifyExportPackage(verifier)
 
```

Comment:
The `autoExport = false` setting allows the CPK to have private packages too. And to be honest, _I don't know_ if we can handle CPKs having private `net.corda` packages - although I suspect the security policy will prevent it :thinking:.

Reply:
Seems like detecting private packages is desired too.

Do you think we could achieve this by looking at the directory paths in the resulting jar? If any file contains the path `/net/corda/` then we output the error?

Reply:
Bnd has already scanned the bundle's metadata. Check the `verifier` object for a `privatePackage` member.

Reply:
Thank you Chris. I'll use that.

Reply:
Done. I've added a test to cover private packages too.